python/pyspark/ml/torch/ssh_manager.py

```diff
@@ -0,0 +1,55 @@
+import os
+import subprocess
+import shutil
+from typing import (List)
+
+from pyspark.ml.torch.utils import write_to_location
+
+class SSHEnvManager:
```

Comment:
I would recommend following the same route as the [HorovodRunner](https://src.dev.databricks.com/databricks/spark-deep-learning-private@360e82a04adbfecafc6b58cf904ef329ac4fbfdd/-/blob/sparkdl/horovod/utils.py?L42) for setting up the environment.

Reply:
+1